README.md

```diff
@@ -1,117 +1,47 @@
 # Chronicle
 
-A tool for recording and querying provenance onto distributed ledgers.
+A tool for recording and querying provenance data stored on distributed ledgers.
 
-## Provenance
-
-![PROV-O](docs/images/starting-points.svg)
-Chronicle implements the
-[PROV-O starting point terms](https://www.w3.org/TR/2013/REC-prov-o-20130430/#description-starting-point-terms)
-, encoding them in
-[JSON-LD compact form](https://json-ld.org/spec/latest/json-ld-api/#compaction)
-onto a backend ledger - currently Sawtooth or an in-memory stub for testing
-purposes.
-
-## Chronicle Vocabulary
-
-As well as core prov, we have additional vocabulary to describe cryptographic
-processes, identities, and prevent conflict between multiple domains on-chain
-using namespaces.
-
-### chronicle:Identity
-
-Represents a cryptographic identity for an agent, supporting a single current
-signing identity via chronicle:hasIdentity and historical identities via
-chronicle:hadIdentity.
-
-### chronicle:publicKey
-
-A value containing a hex encoded ECDSA public key.
-
-Domain: Identity
-
-### chronicle:hasIdentity
-
-The current cryptographic identity of a prov:Agent.
-
-Domain: prov:Agent
-Range: chronicle:Identity
-
-### chronicle:hadIdentity
-
-A historical cryptographic identity for a prov:Agent.
-
-Domain: prov:Agent
-Range: chronicle:Identity
-
-### chronicle:Namespace
+## Getting Started
 
-An IRI containing an external id and uuid part, used for disambiguation. In order
-to work on the same namespace discrete Chronicle instances must share the uuid
-part.
+We recommend you start by exploring the Chronicle examples repo and trying to
+build one of the examples there: <https://github.com/btpworks/chronicle-examples>
```

Comment:
```suggestion
We recommend you start by exploring the
[Chronicle examples repo](https://github.com/btpworks/chronicle-examples)
and experimenting with one of the runnable application domains that it provides.
```
A few thoughts wrapped together in this suggestion, feel free to ignore them all, no blockers with your original.

Reply:
I deliberately chose not to obfuscate the repo URL.

Reply:
Feel free to change the text to something like this:
We recommend you start by exploring the Chronicle examples repo, and trying to
build and experiment with one of the runnable application domains there: <https://github.com/btpworks/chronicle-examples>